src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.io.IOException;
+
+import commands.Command;
+import exceptions.UnsupportedCommandException;
+import format.Ui;
+import tasklist.TaskList;
+
+/**
+ * Represents the Kiwi tasklist app. Named as Duke for legacy purposes.
+ */
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+    private static TaskList taskList;
+    private boolean hasExitCommandBeenSent;
+    private Parser parser;
+
+    /**
+     * Initialise duke with a taskList and a parser.
+     */
+    public Duke() {
+        this.initTaskList();
+        this.initParser();
+    }
+
+    private void initParser() {
+        this.parser = new Parser();
+    }
+
+    /**
+     * Initialise taskList for the app
+     */
+    private void initTaskList() {
+        try {
+            taskList = Storage.setupTaskList();
+        } catch (IOException e) {
+            Ui.print(new String[]{"Something went wrong in loading the task file and parsing", e.getMessage()});
+            taskList = new TaskList();
+        }
+    }
+
+
+    // javafx code adapted/taken from https://se-education.org/guides/tutorials/javaFxPart3.html
+
+
+    /**
+     * This is the main function that generates a response to user inputted commands and strings.
+     *
+     * @param input User input
+     * @return Response after processing the command(s) and argument(s) from the user input
+     */
+    String getResponse(String input) {
```

Comment:
Just wondering, any reason for this to be not public or private and default instead? Usually its preferred to mark methods as private/public unless specifically intended to work with similar classes in the same package.